packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -113,63 +126,77 @@ To complete the one-time verification of the Event Hook:
         * **First Name:** John
         * **Last Name:** Doe
         * **User Name:** john.doe@example.com
+
+1. Navigate back to your local application's console. The following message appears if successful:
+
+    `The user john.doe@example.com has been added to the Okta org!`
+
+1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface from which you can review the complete call response body, header, and other details. Clicking **Replay** on the ngrok inspection interface replays the event hook call to assist in development. See [Replay the Event Hook](#replay-the-event-hook).
+
+    ![A screen shot of the ngrok inspection interface that includes details of a call.](/img/ngrok-and-event-hooks-web-interface-small.png)
+
+### Preview the Event Hook
+
+To run a preview call of your Event Hook:
+
 1. In the Admin Console, go to **Workflow** > **Event Hooks**.
 1. Locate the Event Hook you created during the set-up step. In this example, select `New User Event Hook` or the name you gave the Event Hook.
 1. Click the **Actions** menu for this hook, and select **Preview**.
-1. In the **Configure Event Hook request** section, select an event from the **Event Type** drop-down menu. In this example, there is only one: `User Created (user.lifecycle.create)`.
-1. The most recent event (in this case, user John Doe created above) populates the **Preview & Deliver Event Hook** section with the JSON body of the Event Hook. You can also select an older event from the **System Log Event** drop-down menu. If no event is available, the JSON body populates with sample data.
+1. In the **Configure Event Hook request** section, select an event from the **Event Type** dropdown menu. In this example, there is only one: `User Created (user.lifecycle.create)`.
+1. The most recent event (in this case, user John Doe created previously) populates the **Preview & Deliver Event Hook** section with the JSON body of the Event Hook. You can also select an older event from the **System Log Event** dropdown menu. If no event is available, the JSON body populates with sample data.
 
-    > **Note:** If you are using the preview sample data, you must edit the preview to add a `target` value, including an email using the `alternateId` property. For example, `"target": [{ "alternateId": "john.doe@example.com"}]`. Otherwise, the sample application code throws an error if the value in `target` property is `null`.
+    > **Note:**
+    >
+    > * The preview Event Hook JSON body can be modified for testing or development purposes. Click **Edit** to update the `"target"` field, for example.
+    > * If you are using the preview sample data, you must edit the preview to add a `target` value, including an email using the `alternateId` property. For example, `"target": [{ "alternateId": "john.doe@example.com"}]`. Otherwise, the sample application code throws an error if the value in `target` property is `null`.
```

Comment:
```suggestion
    > * If you are using the preview sample data, you must edit the preview to add a `target` value. Include an email address for the `alternateId` property. For example, `"target": [{ "alternateId": "john.doe@example.com"}]`. Otherwise, the sample application code throws an error if the value in the `target` property is `null`.
```